From: Alexander Graf <agraf@suse.de>
Date: Weg, 20 Mar 2013 15:17:25 +0100
Subject: [PATCH] omap3isp: fix miscompile
Patch-Mainline: No

The omap3isp module requires vm_flags_t. Include the header
that defines it.

Fixes the following breakage:

  drivers/media/platform/omap3isp/ispqueue.h:93:2: error: unknown type name 'vm_flags_t'


Signed-off-by: Alexander Graf <agraf@suse.de>

Index: linux-3.9-rc3-master/drivers/media/platform/omap3isp/ispqueue.h
===================================================================
--- linux-3.9-rc3-master.orig/drivers/media/platform/omap3isp/ispqueue.h
+++ linux-3.9-rc3-master/drivers/media/platform/omap3isp/ispqueue.h
@@ -31,6 +31,7 @@
 #include <linux/mutex.h>
 #include <linux/videodev2.h>
 #include <linux/wait.h>
+#include <linux/mm_types.h>
 
 struct isp_video_queue;
 struct page;
